debian/changelog

```diff
@@ -2,7 +2,13 @@ daos (2.0.0-1) unstable; urgency=medium
   [ Johann Lombardi ]
   * Version bump to 2.0.0
 
- -- Johann Lombardi <johann.lombardi@intel.com>  Mon, 6 Dec 2021 010:00:00 -0100
+ -- Johann Lombardi <johann.lombardi@intel.com>  Fri, 10 Dec 2021 010:00:00 -0100
```

Comment:
Should drop the leading `0`:
```suggestion
 -- Johann Lombardi <johann.lombardi@intel.com>  Fri, 10 Dec 2021 10:00:00 -0100
```

Although, I don't think it's fatal, so in any future update.

Reply:
Good catch. I will fix this at landing time.